pyproject.toml

```diff
@@ -15,6 +15,8 @@ dependencies = [
   # intentionally loose. perhaps these should be vendored to not collide with user code?
   "attrs>=20.1,<24",
   "fastapi>=0.75.2,<0.99.0",
+  # this version specification is pretty arbitrary, and we may not need http2
+  "httpx[http2]>=0.25.0,<0.27",
```

Comment:
what should it be?

Reply:
FWIW, Replicate's Python client uses `httpx>=0.21.0,<1`, and I haven't heard about that causing problems for anyone in the 6 months since we switched to that from requests.